experimental/dds/tree2/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -103,7 +103,16 @@ function invertMark<TNodeChange>(
 ): Mark<TNodeChange>[] {
 	switch (mark.type) {
 		case NoopMarkType: {
-			return [mark];
+			const inverse = { ...mark };
+			if (mark.changes !== undefined) {
+				if (mark.cellId === undefined) {
+					inverse.changes = invertChild(mark.changes, inputIndex);
+				} else {
+					// TODO: preserve modifications to the removed nodes.
+					delete inverse.changes;
```

Comment:
Is this just blocked on refactoring the repair data scheme?

Reply:
Yes.